### DISCUSSION TITLE
結果を表示する画面のhtmlファイルについて

### DISCUSSION THREAD

**Discussion by mitukawa** [zemi-team2/sake#88] [Q&A]
@watagit 

flaskの動作確認用に#66 でhtmlファイルを作りたいのですが、とりあえず、`result.html`で 作成しちゃってもいいですか？
確認したいことは、取得したラベルを`<script>タグ`内で変数に格納する処理の確認で、レイアウトには触れないつもりです。

**Comment by watagit** [comment]
@mitukawa 作成しちゃって問題ないと思われます！ 👍
